build/pipelines/release.yml

```diff
@@ -1,48 +1,492 @@
+        FolderPath: $(System.ArtifactsDirectory)
+        Pattern: Microsoft.WindowsTerminal*.msixbundle
+        UseMinimatch: true
+        signConfigType: inlineSignParams
+        inlineOperation: >-
+          [
+              {
+                  "KeyCode": "Dynamic",
+                  "CertTemplateName": "WINMSAPP1ST",
+                  "CertSubjectName": "CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US",
+                  "OperationCode": "SigntoolSign",
+                  "Parameters": {
+                      "OpusName": "Microsoft",
+                      "OpusInfo": "http://www.microsoft.com",
+                      "FileDigest": "/fd \"SHA256\"",
+                      "TimeStamp": "/tr \"http://rfc3161.gtm.corp.microsoft.com/TSS/HttpTspServer\" /td sha256"
+                  },
+                  "ToolName": "sign",
+                  "ToolVersion": "1.0"
+              },
+              {
+                  "KeyCode": "Dynamic",
+                  "CertTemplateName": "WINMSAPP1ST",
+                  "CertSubjectName": "CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US",
+                  "OperationCode": "SigntoolVerify",
+                  "Parameters": {},
+                  "ToolName": "sign",
+                  "ToolVersion": "1.0"
+              }
+          ]
+    - task: PublishBuildArtifacts@1
+      displayName: 'Publish Artifact: appxbundle-signed'
+      inputs:
+        PathtoPublish: $(System.ArtifactsDirectory)
+        ArtifactName: appxbundle-signed
+
+- ${{ if eq(parameters.buildWPF, true) }}:
+  - job: PackageAndSignWPF
+    strategy:
+      matrix:
+        ${{ each config in parameters.buildConfigurations }}:
+          ${{ config }}:
+            BuildConfiguration: ${{ config }}
+    displayName: Create NuGet Package (WPF Terminal Control)
+    dependsOn: Build
+    steps:
+    - checkout: self
+      clean: true
+      submodules: true
+      persistCredentials: True
+    - task: PkgESSetupBuild@10
+      displayName: Package ES - Setup Build
+      inputs:
+        useDfs: false
+        productName: OpenConsole
+        disableOutputRedirect: true
+    - task: DownloadBuildArtifacts@0
+      displayName: Download x86 PublicTerminalCore
+      inputs:
+        artifactName: wpf-dll-x86-$(BuildConfiguration)
+        itemPattern: '**/*.dll'
+        downloadPath: bin\Win32\$(BuildConfiguration)\
+        extractTars: false
+    - task: DownloadBuildArtifacts@0
+      displayName: Download x64 PublicTerminalCore
+      inputs:
+        artifactName: wpf-dll-x64-$(BuildConfiguration)
+        itemPattern: '**/*.dll'
+        downloadPath: bin\x64\$(BuildConfiguration)\
+        extractTars: false
+    - task: PowerShell@2
+      displayName: Move downloaded artifacts up a level
+      inputs:
+        targetType: inline
+        # Find all artifact files and move them up a directory. Ugh.

```

Comment:
Or was it this guy for the RIP in pepperoni. Either way. I'm still laughing/crying

Reply:
Haha I think it was this one